src/missile.cpp

```diff
@@ -18,7 +18,7 @@ void Missile::draw(GraphicsHandler *graphics) {
         printf("Missile graphics were null! Setting missile graphic...\n");
         set_texture(graphics->load_image("assets/projectiles/missile.png"));
     }
-    graphics->draw(missile_texture_, (int)x_pos(), (int)y_pos());
+    graphics->draw(missile_texture_, (int)x_pos(), (int)y_pos(), GraphicPriority::MIDDLE);
```

Comment:
I like the `GraphicPriority` here. But just as a note: You wanted to know what the next `TODO` should be. Maybe we should work on doing that global config file soon? 
That way we can have the `GraphicPriority::MIDDLE` be something like `GlobalConfig.get(PriorityMissile)` or something along those lines. 

That way, we'll be able to have a config file that we can edit to change all of their layers.